### PR TITLE
Jfrog Anno Mojo maven plugin do not work with jdk8

### PR DESCRIPTION
**Pull request by piddubnyi**
Removed Jfrog Anno Mojo maven plugin and annotations replaced with javadoc attributes.
Solving https://github.com/jcabi/jcabi-beanstalk-maven-plugin/issues/27

### CODE REVIEW

**Comment by dmarkov** [comment]
@piddubnyi Thanks, someone will review your pull request soon

**Comment by dmarkov** [comment]
@pinaf it's yours, please review

File: src/main/java/com/jcabi/beanstalk/maven/plugin/AbstractBeanstalkMojo.java [hunk 1]
```diff
@@ -68,80 +67,57 @@
...
-        required = true,
-        description = "Amazon Elastic Beanstalk configuration template name"
-    )
     private transient String template;
 
     /**
      * WAR file to deploy.
-     * @checkstyle LineLength (3 lines)
+     * @checkstyle LineLength (1 line)
+     * @parameter name="war" defaultValue = "${project.build.directory}/${project.build.finalName}.war"
```
Comment (pinaf) [inline src/main/java/com/jcabi/beanstalk/maven/plugin/AbstractBeanstalkMojo.java]: @piddubnyi isn't it `default-value' ?

Comment (pinaf) [inline src/main/java/com/jcabi/beanstalk/maven/plugin/AbstractBeanstalkMojo.java]: @piddubnyi also, no need for whitspaces around the `=` sign I believe.

**Comment by piddubnyi** [comment]
@pinaf thanks, fixed.

**Comment by pinaf** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by dmarkov** [comment]
@pinaf done, I added **18 mins** in payment `AP-2JC27355ME795612R`, 142 hours and 38 mins spent to complete this

extra minutes for review comments (c=3)

+18 to your rating, your total score is [+6632](http://www.netbout.com/b/35128?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**